packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -245,7 +245,7 @@ export default abstract class Editor extends ObservableMixin() {
 	 *
 	 * See also {@link module:core/editor/editor~Editor.defaultConfig}.
 	 */
-	public static builtinPlugins?: Array<PluginConstructor<Editor>>;
+	public static builtinPlugins?: ReadonlyArray<PluginConstructor<Editor>>;
```

Comment:
I'm not sure if this will not break the ability to extend builds? For example like here https://github.com/ckeditor/ckeditor5/blob/29f10a3b67026074b9c8475bec8dc69cb2288f58/docs/_snippets/build-classic-source.js#L12-L14